server/service/service.go

```diff
@@ -0,0 +1,95 @@
+package service
+
+import (
+	"context"
+
+	"github.com/openconfig/bootz/proto/bootz"
+	"github.com/openconfig/gnmi/errlist"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type EntityLookup struct {
```

Comment:
What is the definition of an entity here? (I think this might be a gap in my understanding of bootz.) Are we saying that an entity is a device, or it's just something that can run bootz?

Reply:
control card or chassis

Reply:
Suggest to add a comment that just explains this briefly.